src/EnergyPlus/HeatRecovery.cc

```diff
@@ -364,107 +277,101 @@ namespace HeatRecovery {
-                                  NodeInputManager::CompFluidStream::Primary,
-                                  ObjectIsNotParent);
-            state.dataHeatRecovery->ExchCond(ExchNum).SecInletNode =
-                GetOnlySingleNode(state,
-                                  state.dataIPShortCut->cAlphaArgs(7),
-                                  ErrorsFound,
-                                  DataLoopNode::ConnectionObjectType::HeatExchangerAirToAirFlatPlate,
-                                  state.dataIPShortCut->cAlphaArgs(1),
-                                  DataLoopNode::NodeFluidType::Air,
-                                  DataLoopNode::ConnectionType::Inlet,
-                                  NodeInputManager::CompFluidStream::Secondary,
-                                  ObjectIsNotParent);
-            state.dataHeatRecovery->ExchCond(ExchNum).SecOutletNode =
-                GetOnlySingleNode(state,
-                                  state.dataIPShortCut->cAlphaArgs(8),
-                                  ErrorsFound,
-                                  DataLoopNode::ConnectionObjectType::HeatExchangerAirToAirFlatPlate,
-                                  state.dataIPShortCut->cAlphaArgs(1),
-                                  DataLoopNode::NodeFluidType::Air,
-                                  DataLoopNode::ConnectionType::Outlet,
-                                  NodeInputManager::CompFluidStream::Secondary,
-                                  ObjectIsNotParent);
-
-            TestCompSet(state,
-                        cHXTypes(state.dataHeatRecovery->ExchCond(ExchNum).ExchTypeNum),
-                        state.dataHeatRecovery->ExchCond(ExchNum).Name,
-                        state.dataIPShortCut->cAlphaArgs(5),
-                        state.dataIPShortCut->cAlphaArgs(6),
-                        "Process Air Nodes");
+                break;
+            }
+            thisExchanger.hARatio = state.dataIPShortCut->rNumericArgs(1);
+            thisExchanger.NomSupAirVolFlow = state.dataIPShortCut->rNumericArgs(2);
+            thisExchanger.NomSupAirInTemp = state.dataIPShortCut->rNumericArgs(3);
+            thisExchanger.NomSupAirOutTemp = state.dataIPShortCut->rNumericArgs(4);
+            thisExchanger.NomSecAirVolFlow = state.dataIPShortCut->rNumericArgs(5);
+            thisExchanger.NomSecAirInTemp = state.dataIPShortCut->rNumericArgs(6);
+            thisExchanger.NomElecPower = state.dataIPShortCut->rNumericArgs(7);
+            thisExchanger.SupInletNode = GetOnlySingleNode(state,
+                                                           state.dataIPShortCut->cAlphaArgs(5),
+                                                           ErrorsFound,
+                                                           DataLoopNode::ConnectionObjectType::HeatExchangerAirToAirFlatPlate,
+                                                           state.dataIPShortCut->cAlphaArgs(1),
```

Comment:
`thisExhanger.Name`

Reply:
Replaced a chunk of these with thisExchanger.Name